wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -13,7 +13,9 @@ end
 
 namespace :wagn do
   desc "create a wagn database from scratch"
-  task :create do
+  task :recreate do
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
+
     puts "dropping"
     #fixme - this should be an option, but should not happen on standard creates!
     begin
```

Comment:
This is where I wanted to change the old 'create' task to be called 'recreate' because that better describes what it does, but I had to change it back so create is the original drop, load, re-populate task.  This is the wrong name for the new task which clears tables and re-populates.  I think that is a valid task to want, but the names are not right yet.